latte.go

```diff
@@ -5,7 +5,7 @@ type latte struct{}
 
 var _ Theme = latte{}
 
-func (latte) Name() string { return "latte" }
+func (latte) Name() string { return "Latte" }
```

Comment:
Same comment as above.
```suggestion
func (latte) Name() string { return "latte" }
```